descartes_light/solvers/include/descartes_light/solvers/bgl/bgl_dijkstra_solver.h

```diff
@@ -0,0 +1,43 @@
+#ifndef DESCARTES_LIGHT_SOLVERS_BGL_BGL_DIJKSTRA_SOLVER_H
+#define DESCARTES_LIGHT_SOLVERS_BGL_BGL_DIJKSTRA_SOLVER_H
+
+#include <descartes_light/solvers/bgl/bgl_solver.h>
+
+namespace descartes_light
+{
+/**
+ * @brief BGL solver implementation that constructs vertices and edges in the build function and uses Dijkstra's
+ * algorithm with a default visitor to search the graph
+ */
+template <typename FloatType>
+class BGLDijkstraSolverVE : public BGLSolverBaseVE<FloatType>
```

Comment:
Since this is a full solver is the post fix `VE` needed?

Reply:
I added it to mean that `build` will create both vertices (V) and edges (E). I think we should still have this suffix because our next implementation will be a solver that only adds vertices and uses a different custom visitor to add edges dynamically during the search

Reply:
I'm open to suggestions on how to communicate this difference more effectively. I just haven't been able to think of a better name that overly long

Reply:
For the record, we append the BGL class names with the acronyms SVSE (static vertex static edge), SVDE (static vertex dynamic edge), and DVDE (dynamic vertex dynamic edge) to reflect whether the vertices/edges get constructed in `build` (i.e. static) or in the search (i.e. dynamic)